src/js/modules/articles/components/SearchResults.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { compose } from "redux";
+import { graphql } from "react-apollo";
+import gql from "graphql-tag";
+import humps from "humps";
+import { Row, Col } from "react-bootstrap";
+import injectSheet from "react-jss";
+
+import { TallAd } from "../../advertisements/components";
+import ArticleList from "./ArticleList";
+import { PropagateLoader } from "../../core/icons";
+
+const styles = {
+  loaderContainer: {
+    height: "6px",
+    margin: "20px auto",
+    textAlign: "center",
+    width: "1px",
+  },
+  articleList: {
+    paddingRight: "14px !important",
+  },
+  tallAdContainer: {
+    paddingLeft: "14px !important",
+    borderLeft: "solid 1px #ddd",
+  },
+};
+
+const SearchResultsQuery = gql`
+  query SearchResultsQuery($query: String!) {
+    searchArticles(query: $query) {
+      searchable {
+        id
+        title
+        summary
+        slug
+        created_at
+        media {
+          attachment_url
+          title
+        }
+        section {
+          permalink
+        }
+        contributors {
+          first_name
+          last_name
+        }
+      }
+    }
+  }
+`;
+
+const SearchResults = ({ classes, data }) => {
+  if (data.loading) {
+    return (
+      <div className={classes.loaderContainer}>
+        <PropagateLoader />
```

Comment:
Not a very clear name. What is it propagating?

Reply:
And if that's a visual thing, I'd avoid naming components after how they look. It's tempting, but then it's not obviously apparent what the component is when you're just looking at the codebase